src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -248,8 +249,12 @@ def _on_focussed_view(self, focussed_view):
         for viewport in self.tab_data_model.viewports.value:
```

Comment:
I *think* this whole function could be moved into the viewport itself (FastEMMainViewport). By overriding .SetFocus(), each viewport will know when it becomes the focused view(port) or not. This would avoid the (explicit) `for` loop, and the `if` would become just `if focus`.

Reply:
Nice suggestion, thanks!